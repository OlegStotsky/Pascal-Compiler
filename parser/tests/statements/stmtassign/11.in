var
	a : integer;
	b : integer;

begin
	a[1] := 3;
end;