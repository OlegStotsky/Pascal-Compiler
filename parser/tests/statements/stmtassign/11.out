Error at line 6, column 6 : array expected but integer found
